Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/main.kt

```diff
@@ -191,8 +193,17 @@ private fun findFilesByGlobs(roots: List<Path>, globs: List<String>): Map<Path,
     return relativeToRoot
 }
 
+
+
 private fun processCLib(flavorName: String, cinteropArguments: CInteropArguments,
                         additionalArgs: InternalInteropOptions): Array<String>? {
+
+    // TODO: Replace this with a commandline-supplied file through cinterop
+    val jarfile = JarFile("/Users/odowa/Code/example/Foo.jar")
```

Comment:
Next CL will probably be on adding support for different Jar files. Tests are currently fixed to this jar file that supplies the bindings to the ObjC functions.

Reply:
This is done. https://github.com/LibanOdowa/kotlin-native/pull/6